src/reusable/button.js

```diff
@@ -59,26 +59,22 @@ const getDisabledCSS = (disabled) => {
 };
 
 const getButtonCSS = (kind, disabled) => {
-  const color = getButtonColor(kind);
+  const color = getButtonBackground(kind);
 
   if (kind === 'reset') {
     return {};
   }
-
   return {
     alignItems: 'center',
     borderRadius: '2px',
-    color:
-      lightOrDark(color['400']) === 'light' || kind === 'subtle'
-        ? 'inherit'
-        : 'white',
+    color: kind === 'tertiary' || kind === 'primary' || kind === 'subtle' ? 'inherit' : 'white',
```

Comment:
```suggestion
    color: ['tertiary', 'primary', 'subtle'].includes(kind) ? 'inherit' : 'white',
```

Not a blocker, but I wanted to show a simpler way to write the same logic.